### DISCUSSION TITLE
[Feature request] Repository private notes

### DISCUSSION THREAD

**Discussion by eduherminio** [community/community#60810] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Feature request: being able to save a small note about any starred repository.
- They shouldn't be deleted if a repository changes lists.
- Ideally those notes would be searchable.

Having private, searchable tags instead of notes could be another suggestion to facilitate stars classification.

**Comment by ribeiromarilene** [comment]
[]()
